src/resonanceReconstruction/src/channelRadius.hpp

```diff
@@ -2,9 +2,5 @@ inline auto channelRadius( double atomicWeightRatio ){
   constexpr double amu = 1.00866491582;
   const auto scalar =
     0.123 * std::pow( amu * atomicWeightRatio, 1. / 3. ) + 0.08;
-  
-  return
-    [ radius = scalar * rootBarn ]
-    ( Quantity< ElectronVolts > /* energy */ )
-    { return radius; };
+  return radius(scalar);
 }
```

Comment:
Just this change made the compilation time drop by more than half? I'm trying to understand why, since radius() does exactly the same thing.

Reply:
Each returned a lambda that did exactly the same thing. But when they're returned separately, they're different lambdas, from a technical language standpoint, which means that some template construct somewhere might have been instantiated for both lambdas individually - because they're not seen as being the same, even though WE know that they do the same thing. If the instantiated stuff is expensive, then, well, there's the problem.

Reply:
And this is why I think that consolidating this code's considerable repetitive material is likely to trigger further improvements in compile times.

Reply:
Importantly, also: the ability to use the ternary conditional in changes 2 and 3 *required* that change 1 be made first, so that the two possibilities selected by the ternary conditional were of the same type.